openedx/features/course_duration_limits/config.py

```diff
@@ -17,3 +24,36 @@
     flag_name=u'studio_ui',
     flag_undefined_default=False
 )
+
+EXPERIMENT_ID = 11
+EXPERIMENT_DATA_HOLDBACK_KEY = 'holdback_{0}'
+
+
+@receiver(ENROLL_STATUS_CHANGE)
+def set_value_for_content_type_gating_holdback(sender, event=None, user=None, **kwargs):  # pylint: disable=unused-argument
+    experiment_data_holdback_key = EXPERIMENT_DATA_HOLDBACK_KEY.format(user)
+    if event == EnrollStatusChange.enroll:
+        try:
+            ExperimentData.objects.get(
```

Comment:
Probably both better and clearer to use `ExperimentData.objects.filter(...).exists()` for this check.